config/settings.py

```diff
@@ -314,3 +320,24 @@
 AWS_TRANSLATE_ACCESS_KEY = env('AWS_TRANSLATE_ACCESS_KEY')
 AWS_TRANSLATE_SECRET_KEY = env('AWS_TRANSLATE_SECRET_KEY')
 AWS_TRANSLATE_REGION = env('AWS_TRANSLATE_REGION')
+
+
+# Email Configuration
+AWS_ENVIRONMENT = env('AWS_ENVIRONMENT')
+DEFAULT_FROM_EMAIL = env('DEFAULT_FROM_EMAIL')
+
+# Celery config
+BROKER_URL = env("REDIS_URL")
+CELERY_TIMEZONE = env('TIME_ZONE')
+CELERY_TASK_TRACK_STARTED = True
+CELERY_TASK_TIME_LIMIT = 30 * 60
+
+
+if DEBUG:
+    EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+elif not AWS_ENVIRONMENT:  # if deployed environment is other than aws
+    EMAIL_BACKEND = 'django_ses.SESBackend'
+    AWS_S3_ACCESS_KEY_ID = env('AWS_S3_ACCESS_KEY_ID')
+    AWS_S3_SECRET_ACCESS_KEY = env('AWS_S3_SECRET_ACCESS_KEY')
+else:   # if deployed environment is aws
+    EMAIL_BACKEND = 'django_ses.SESBackend'
```

Comment:
Let's not use AWS_S3 here as this is AWS_SES,

Let's do something like this
```suggestion
if env('USE_AWS_SES'):
    EMAIL_BACKEND = 'django_ses.SESBackend'
    AWS_SES_REGION_NAME = env('AWS_SES_REGION_NAME')
    AWS_SES_ACCESS_KEY_ID = env('AWS_SES_ACCESS_KEY_ID')
    AWS_SES_SECRET_ACCESS_KEY = env('AWS_SES_SECRET_ACCESS_KEY')
else:
    EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
```